pkgs/applications/networking/cloudflare-dyndns/default.nix

```
@@ -11,7 +11,7 @@
 , requests
 }:
 
-buildPythonApplication rec {
+buildPythonPackage rec {
```

This is a client and not so much a module. I would like to suggest to keep it as application (`buildPythonApplication`) but update to the stype which is preferred nowadays.

```suggestion
python3.pkgs.buildPythonApplication rec {
```

`all-packages.nix` should be updated.

```nix
  cloudflare-dyndns = callPackage ../applications/networking/cloudflare-dyndns { };
```